File: Java/49. Group Anagrams.java
class Solution {
    public List<List<String>> groupAnagrams(String[] strs) {
       String[] result; 
        for(String s : strs){
            char[] sArray = s.toCharArray();
            Arrays.sort(sArray);
            Arrays.toString(sArray);
        }

    }
}

File: C/49. Group Anagrams.cpp
class Solution {
public:
    vector<vector<string>> groupAnagrams(vector<string>& strs) {
        unordered_map<string, vector<string>> map;

        for (int i = 0; i < strs.size(); i++) {
            string sortedStr = strs[i];
            sort(sortedStr.begin(), sortedStr.end());
            map[sortedStr].push_back(strs[i]);
        }

        vector<vector<string>> result;
        for (auto it = map.begin(); it != map.end(); it++) {
            result.push_back(it->second);
        }

        return result;
    }
};
